plugins/woocommerce-blocks/assets/js/data/cart/thunks.ts

```diff
@@ -194,10 +199,12 @@ export const removeCoupon =
  *
  * @param {number} productId    Product ID to add to cart.
  * @param {number} [quantity=1] Number of product ID being added to cart.
+ * @param {array}  [variation] Array of variation attributes for the product.
+ * @param {Object}  [additionalData] Array of additional fields for the product.
  * @throws           Will throw an error if there is an API problem.
  */
 export const addItemToCart =
-	( productId: number, quantity = 1 ) =>
+	( productId: number, quantity = 1, variation: Variation[], unknown: additionalData: Record<string, unknown> = {} ) =>
```

Comment:
```suggestion
	( productId: number, quantity = 1, variation: Variation[], additionalData: Record<string, unknown> = {} ) =>
```
May of been a copy paste error, but don't need the additional `unknown` here.